source/stylesheets/refills/_progress-bar.scss

```diff
@@ -1,88 +1,67 @@
 .progress-bar {
-  $base-border-color: gainsboro !default;
-  $base-background-color: white !default;
-  $base-border-radius: 3px !default;
-  $action-color: #477DCA !default;
-  $base-link-color: $action-color !default;
-  $progress-border-color: $base-border-color;
-  $progress-border: 1px solid $progress-border-color;
-  $progress-meter-border-color: $action-color;
-  $progress-meter-border: 1px solid darken($progress-meter-border-color, 15%);
-  $progress-meter-color: $progress-meter-border-color;
-  $progress-background: darken($base-background-color, 5);
+  $base-background-color: #fff !default;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets